test_common/helpers/setup.rb

```diff
@@ -175,8 +175,25 @@ def answer_make_employer_contract_claim_question
         employers_contract_claim_page.make_employer_contract_claim_question.set_for(user)
       end
 
-      def upload_additional_information
-        employers_contract_claim_page.upload_additional_information.set(user.additional_information)
+      # Additional Information Page
+      def answer_upload_additional_information_question
+        # Source: https://stackoverflow.com/questions/32880524/how-do-you-test-uploading-a-file-with-capybara-and-dropzone-js
+
+        # Generate a fake input selector
+        page.execute_script <<-JS
```

Comment:
This sort of stuff should really be in a page object.  In general page objects should wrap all the selenium type stuff.  Could you look into refactoring this a little or if its too time consuming, create a ticket to do it later ?

The other advantage is that if this is in a method in a page object, it is clear of its intentions by the method name - for example - it could be 'attach_additional_info_file'.

Reply:
Refactored in 4404fc2